client/templates/helpers.js

```diff
@@ -0,0 +1,13 @@
+Template.registerHelper('displayOrderNumber', (order) => {
+  console.log(order);
+  if (order.shopifyOrderId) {
+    return '<a href="http://getoutfitted.myshopify.com/admin/orders/'
+    + order.shopifyOrderId
+    + '">Order #' + order.shopifyOrderNumber + '</a>';
+  } else if (order.shopifyOrderNumber) {
+    return 'Order #' + order.shopifyOrderNumber;
+  }
+
+  // Default
+  return 'Order #' + order._id;
+});
```

Comment:
@spencern what does Template.registerHelper do? Does that make it a global that all templates can use?